pkg/render/applicationlayer/applicationlayer.go

```diff
@@ -330,6 +335,11 @@ func (c *component) containers() []corev1.Container {
 						MountPath: ModSecurityRulesetVolumePath,
 						ReadOnly:  true,
 					},
+					{
```

Comment:
The order matters a lot here. Maybe a comment would be good or a test about the order. Comment in the test maybe?